frontend/device/android/device.lua

```diff
@@ -375,7 +377,8 @@ function Device:canExecuteScript(file)
     end
 end
 
-android.LOGI(string.format("Android %s - %s (API %d) - flavor: %s",
-    android.prop.version, getCodename(), Device.firmware_rev, android.prop.flavor))
+android.LOGI(string.format("Android %s - %s (API %d, product %s) - flavor: %s",
```

Comment:
That's not needed. android.prop.product is == Device.model and that's already logged on a bunch of sites.